### PR TITLE
Use posixpath.sep for upload keys

### PR DESCRIPTION
**Pull request by kevinschaul**
Fixes upload URLs for Windows, e.g. js\base.js -> js/base.js

### CODE REVIEW

File: s3peat/__init__.py [hunk 1]
```diff
@@ -181,6 +182,8 @@ def _key(self, filename):
 
         # Strip the filename of leading path separators
         filename = filename.lstrip(os.path.sep)
+        # Replace path separators with posix separator
+        filename = filename.replace(os.path.sep, posixpath.sep)
         # Join it to the prefix and go!
         return '/'.join((self.prefix, filename))
```
Comment (shakefu) [inline s3peat/__init__.py]: Should this join use posixpath.sep as well?

Comment (kevinschaul) [inline s3peat/__init__.py]: I have no preference

**Comment by shakefu** [comment]
Looks good I'll merge and release soon as I get back to my computer.

**Comment by shakefu** [comment]
This is out in 0.5.1.

**Comment by kevinschaul** [comment]
Awesome. Thank you.

**Pull request merged**